app/templates/suppliers/duns_number.html

```diff
@@ -18,6 +18,17 @@
 
 {% block main_content %}
 
+{% if form.duns_number.errors[0] == 'DUNS number already used' %}
+<div class="validation-masthead" role="group" aria-labelledby="validation-masthead-heading">
+    <h1 class="validation-masthead-heading" id="validation-masthead-heading">
+        A supplier account already exists with that DUNS number
```

Comment:
Also, the copy of the long message at the top is likely to change when Roz gets back. The short message is more likely to be OK as it is I think.